distribution/smarthome/conf/smarthome.cfg

```diff
@@ -18,3 +18,5 @@ org.eclipse.smarthome.threadpool:discovery=3
 
 # Non-scheduled thread pools can also provide a max size
 org.eclipse.smarthome.threadpool:safeCall=3,10
+
+org.eclipse.smarthome.binding.sonos:opmlUrl=http://opml.radiotime.com/Describe.ashx?c=nowplaying&partnerId=IAeIhU42&id=%id&serial=%serial
```

Comment:
Please remove this line - this was the whole exercise about the opml url: Not having any external url referenced from Eclipse SmartHome. We will add this setting instead to openHAB 2.